### DISCUSSION TITLE
[ FF7 ] Black screen (no crash) after using cable car to Gold Saucer with FFNx 1.17.1

### DISCUSSION THREAD

**Discussion by cassiuskad** [julianxhokaxhiu/FFNx#645] [Q&A] [answered]
**Describe the bug**
After taking the cable car in North Corel, the cinematic plays and the screen stays black when you are supposed to arrive at Gold Saucer. The music is still running, so the game does not crash.

**To Reproduce**
Go to North Corel and take the cable car to Gold Saucer

**GPU (please complete the following information):**
- RTX 2070S
- 546.65

**Additional context**
Reverting to FFNx 1.17.0 fixes the issue

**Comment by eve-atum** [comment, marked as answer]
Please make sure your MSAA is set no higher than 2x, and internal resolution scaler is automatic.  Your symptoms are common for someone trying to push their graphics card beyond what it's capable of.

**Comment by cassiuskad** [comment]
> Please make sure your MSAA is set no higher than 2x, and internal resolution scaler is automatic. Your symptoms are common for someone trying to push their graphics card beyond what it's capable of.

This works, thank you!

**Comment by pillowtrucker** [comment]
uh are you sure this isn't a bug? This happens on a 3080 too
